pkg/armhelpers/utils/util.go

```diff
@@ -100,6 +105,17 @@ func VmssNameParts(vmssName string) (poolIdentifier, nameSuffix string, err erro
 	return vmssNameParts[vmssAgentPoolNameIndex], vmssNameParts[vmssClusterIDIndex], nil
 }
 
+// WindowsVmssNameParts returns the agent pool name from the VMSS name.
+// For windows, the VMSS name is 'aks' concatenated with agentpool name
+func WindowsVmssNameParts(vmssName string) (poolIdentifier string, err error) {
```

Comment:
can this be unit tested?

Reply:
added a simple unit test for WindowsVmssNameParts.